3party/protobuf/CMakeLists.txt

```diff
@@ -25,6 +24,14 @@ set(SRC
   protobuf/src/google/protobuf/wire_format_lite.cc
 )
 
+if (CMAKE_SYSTEM_PROCESSOR MATCHES "([xX]86)|([aA][mM][dD]64)")
```

Comment:
Can CMAKE_SYSTEM_PROCESSOR be a lower case?

Reply:
Why is this check needed?
It looks like it could be removed

Reply:
Yeah
https://github.com/organicmaps/protobuf/blob/9442c12e866b56369f1c53abea1261259c924a19/cmake/libprotobuf-lite.cmake#L11-L12